lib/eslint.js

```diff
@@ -801,10 +798,7 @@ module.exports = (function() {
             }
 
             parseResult = parse(
-                stripUnicodeBOM(text).replace(/^#!([^\r\n]+)/, (match, captured) => {
-                    shebang = captured;
-                    return `//${captured}`;
-                }),
+                stripUnicodeBOM(text).replace(/^#!([^\r\n]+)/, (match, captured) => `//${captured}`),
```

Comment:
This will fail to parse a shebang that contains `\u2028` or `\u2029`. For example, Node is able to execute the following file without any problems, but eslint throws a parsing error. Note that there is a `\u2028` character between `bin` and `/env`.

```js
#!/usr/bin /env node
console.log("foo")
```

Actually, it seems like this was a problem before this PR, too. If there's no easy solution to this, we can probably just leave it.